### PR TITLE
Fix sso login. Reset cookie for sso for openSAP.

### LINKED ISSUE #344: Login via single sign-on leads to an undefined state.
Although the login was successful, the login button is shown.
When opening the single sign-on view, cookies have to be deleted. Otherwise login view will be skipped and the mobile website is presented to the user.

### PR DESCRIPTION
**Pull request by mathebox**
- Send notification for login state change.
- Correct handling of network indicator animation. It would spin indefinitely.

Fixes #344

### CODE REVIEW

**Comment by mathebox** [comment]
This will not reset the cookies of safari.

File: xikolo-ios/ViewControllers/Main/LoginViewController.swift [hunk 1]
```diff
@@ -43,21 +43,23 @@ class LoginViewController : AbstractLoginViewController, WKUIDelegate {
...
-//        wkViewController.load(request)
+        self.performSegue(withIdentifier: "ShowSSOWebView", sender: self)
     }
 
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
         if segue.identifier == "ShowSSOWebView" {
             let vc = segue.destination as! WebViewController
             vc.url = Routes.SSO_URL
+
+            #if OPENSAP
```
Comment (jprberlin) [inline xikolo-ios/ViewControllers/Main/LoginViewController.swift]: openWHO has SSO as well...

Comment (mathebox) [reply xikolo-ios/ViewControllers/Main/LoginViewController.swift]: Should we reset the cookies for OpenWHO as well?

Comment (jprberlin) [reply xikolo-ios/ViewControllers/Main/LoginViewController.swift]: yes, i guess we should delete all cookies given that this in the scope of  the web view and the app only

**Pull request merged**